qucs/qucs/octave_window.cpp

```diff
@@ -119,7 +113,30 @@ bool OctaveWindow::startOctave()
   // append process PATH, othewise Octave does not find gnuplot
   QProcessEnvironment env = QProcessEnvironment::systemEnvironment();
   env.insert("PATH", env.value("PATH") + sep + QucsSettings.BinDir );
+#ifdef __MINGW32__
+  env.insert("PATH", env.value("PATH") + sep + QucsSettings.OctaveBinDir.canonicalPath() );
```

Comment:
this specializes line 115 for a particular platform/compiler. line 115 is wrong already, better remove it.

`PATH` is an environment variable. should be taken from the environment. customizing the environment does not (should not, must not) require patching qucs here.